tests/acceptance/welcome/payment-info-test.js

```diff
@@ -0,0 +1,84 @@
+import Ember from 'ember';
+import startApp from '../../helpers/start-app';
+import { stripeMock } from '../../../utils/stripe';
+import { stubRequest } from "../../helpers/fake-server";
+
+var application;
+
+module('Acceptance: WelcomePaymentInfo', {
+  setup: function() {
+    application = startApp();
+  },
+  teardown: function() {
+    stripeMock.createStripeToken = null;
+    Ember.run(application, 'destroy');
+  }
+});
+
+test('visiting /welcome/payment-info when not logged in', function() {
+  visit('/welcome/payment-info');
+
+  andThen(function() {
+    equal(currentPath(), 'login');
+  });
+});
```

Comment:
after #48 is merged there will be a `expectRequiresAuthentication` helper that we can use here